libraries/MySensors/examples/DustSensor/DustSensor_SamYoung_DSM501.ino

```diff
@@ -0,0 +1,121 @@
+/*
+  Arduino Dust Sensor for SamYoung DSM501
+  connect the sensor as follows :
+          Pin 2 of dust sensor PM1      -> Digital 3 (PMW)
+	  Pin 3 of dust sensor          -> +5V 
+	  Pin 4 of dust sensor PM2.5    -> Digital 6 (PWM) 
+	  Pin 5 of dust sensor          -> Ground
+  Datasheet: http://www.samyoungsnc.com/products/3-1%20Specification%20DSM501.pdf
+  Contribution: epierre
+  
+  The dust sensor used (see purchase guide for latest link):
+ 
+  
+*/
+
+#include <MySensor.h>  
+#include <SPI.h>
+
+#define CHILD_ID_DUST_PM10            0
+#define CHILD_ID_DUST_PM25            1
+#define DUST_SENSOR_DIGITAL_PIN_PM10  6
+#define DUST_SENSOR_DIGITAL_PIN_PM25  3
+
+unsigned long SLEEP_TIME = 30*1000; // Sleep time between reads (in milliseconds)
+//VARIABLES
+int val = 0;           // variable to store the value coming from the sensor
+float valDUSTPM25 =0.0;
+float lastDUSTPM25 =0.0;
+float valDUSTPM10 =0.0;
+float lastDUSTPM10 =0.0;
+unsigned long duration;
+unsigned long starttime;
+unsigned long endtime;
+unsigned long sampletime_ms = 30000;
+unsigned long lowpulseoccupancy = 0;
+float ratio = 0;
+long concentrationPM25 = 0;
+long concentrationPM10 = 0;
+
+MySensor gw;
+MyMessage dustMsgPM10(CHILD_ID_DUST_PM10, V_DUST_LEVEL);
```

Comment:
This is called V_LEVEL now. 
And if possible send a texual V_UNIT_PREFIX (e.g. "ppm").